app/Keycloak/Listeners/EnableClients.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Keycloak\Listeners;
+
+use App\Domain\Integrations\Events\IntegrationActivated;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Keycloak\Exception\KeyCloakApiFailed;
+use App\Keycloak\Repositories\KeycloakClientRepository;
+use App\Keycloak\Service\ApiClient;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Psr\Log\LoggerInterface;
+use Throwable;
+
+final class EnableClients implements ShouldQueue
```

Comment:
I think you missed the linking of this new listener? This should have been done in another pull request?

Reply:
you are right, there currently is no action to re-enable all clients.
This regarding the ticket we discussed this morning.
I will park the enable code in a seperate draft pr

Reply:
If this is a new class it also lacks some tests.